dockerfiles/che/entrypoint.sh

```diff
@@ -306,7 +306,7 @@ add_cert_to_truststore() {
   echo "$1" > $SELF_SIGNED_CERT
 
   # make sure that owner has permissions to write and other groups have permissions to read
-  chmod 644 $JAVA_TRUST_STORE
+  chmod 644 $JAVA_TRUST_STORE || true
```

Comment:
should it be true or a message should be logged ?

Reply:
```
Using custom assembly from /home/user/codeready
Found a custom cert. Adding it to java trust store /home/user/cacerts
chmod: changing permissions of '/home/user/cacerts': Operation not permitted
```

I suspect even when I use `|| true` we'll get that in the log... we'll just not abort the entrypoint.sh script at that point.